planet/http.py

```diff
@@ -136,8 +137,9 @@ async def araise_for_status(*args, **kwargs):
         self._client.event_hooks['response'] = [
             alog_response, araise_for_status
         ]
-        self.retry_wait_time = RETRY_WAIT_TIME
-        self.retry_count = RETRY_COUNT
+
+        self.max_retries = MAX_RETRIES
```

Comment:
Conversation with @sgillies revealed it may be beneficial to have `max_retries` and `max_retry_backoff` be instance states, so I restored them after briefly removing them.